### DISCUSSION TITLE
Show basic fields for a form, with option to expand advanced fields

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#12946] [Help]
### Package

Form builder

### Package Version

v3.2.82

### How can we help you?

On my settings page, I would like to show only basic fields but let the user expand and show advanced fields.

`collapsible` from the section seems to be a all or nothing mechanism, any idea how I can achieve the above?

**Comment by dissto** [comment]
Perhaps the [Accordion](https://filamentphp.com/plugins/lara-zeus-accordion) plugin is what you want.

If you dont want to use a plugin. One way that comes to mind is to have a toggle that would [toggle the visibility](https://filamentphp.com/docs/3.x/forms/fields/getting-started#hiding-a-field) of other components you want hidden (or to be advanced) 🤔

**Reply by bilogic** [reply]
Hmm, will the toggle clear out the unsaved data that the user has already populated?

**Reply by dissto** [reply]
No, just the visibility. But you may also need take care of [dehydration](https://filamentphp.com/docs/2.x/forms/advanced#dehydration) in case "advanced" was toggled of and they had already entered data prior.

**Reply by bilogic** [reply]
I kind of have an idea, but do you know if there any examples anywhere I can fiddle around with? Thanks!

**Reply by dissto** [reply]
Yea there is an example in the docs: 😊
https://filamentphp.com/docs/3.x/forms/advanced#conditionally-hiding-a-field
